tests/views/test_ensemble_selector.py

```diff
@@ -83,3 +83,33 @@ def test_ensemble_refresh(
         dash_duo, "#" + plugin.uuid("ensemble-multi-selector")
     )
     assert refreshed_cases == expected_cases
+
+
+def test_ensemble_color(mock_data, dash_duo, tmpdir):
```

Comment:
doesn't matter i guess  - but `tmpdir` is not used as far as i can see?

Reply:
Good catch! it is there from a failed attempt to run the test in a temp folder and copy the `ert-style.css` file there. But that did not seem to work for some reason and I abandoned that route. 

I will remove it before merge